resources/lang/es.json

```diff
@@ -0,0 +1,24 @@
+{
+    "Welcome to VoteLocalKS.org": "Bienvenido a VoteLocalKS.org",
+    "Get text message notifications with relevant information about local elections in Kansas.": "Recibe mensajes de texto con información importante sobre nuestras elecciones en Kansas.",
+    "Get started now!": "¡Empieza ahora!",
+    "Learn more": "Aprende más",
+    "Step": "Paso",
+    "Click here": "Haga clic aquí",
+    "to provide your number and start your subscription (100% free).": "para proporcionar su número y comenzar su suscripción (100% gratis).",
+    "Get up to date news about KS voter registration, elections, candidates and&nbsp;more.": "Mantente informado en cosas como registrarse para votar en Kansas, elecciones, candidatos y&nbsp;más",
+    "Stay locally active and engaged. It couldn’t be&nbsp;easier!": "¡Nunca ha sido más fácil estar activo e involucrado en el&nbsp;proceso!",
+    "START": "INICIAR",
+    "Available in Spanish": "¡Disponible en Español!",
+    "Log in": "Inicie sesión",
+    ":login to manage your language preferences!": "¡:login para administrar sus preferencias de idioma!",
+    "Project Contributors": "Colaboradores del proyecto",
+    "These people have been involved in the creation and maintenance of the Vote Local platform.": "Estas personas han estado involucradas en la creación y mantenimiento de la plataforma de Vote Local.",
+    "Feel like helping out? Check out our list of": "¿Tienes ganas de ayudar? Consulte nuestra lista de",
+    "issues": "problemas",
+    "on our": "en nuestro",
+    "Brought to you by the&nbsp;community": "Presentado por la&nbsp;comunidad",
+    "Vote Local is created and operated by local community organizations :kmuw, :devict, and :abc.": "Vote Local es creado y operado por organizaciones comunitarias locales :kmuw, :devict, y :abc.",
```

Comment:
This one doesn't seem to be getting translated to Spanish when viewing the `/es` version of the homepage. Any idea why?

Reply:
Fixed!